src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Layout from './components/layout/Layout';
+import SignIn from './components/signin/SignIn';
+import Order from './components/order/Order';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+      <Routes>
+        <Route path='/' element={<Layout />}>
+          <Route path='' element={<Order />}/>
+          <Route path='login' element={<SignIn />} />
```

Comment:
How can we made a route without `/` in path here?

Reply:
As I can recall, without `/` in path. It creates a relative route means that if any path ends with `/login`, React will render SignIn component.